### ISSUE TITLE
Is there an appetite for a docs review?

### ISSUE THREAD

**Issue by jdevera** [criteo/command-launcher#141]
In my use of the command launcher, I create packages which I then host and I use the automatic update functionalities. To get there, I've been an avid consumer of the documentation. And it has been enough, since I did get it working, but I felt at times that I had to _fight_ it a bit.

I thought I could perhaps contribute back by doing a light review of the docs, perhaps adding some things that I had to figure out from experimentation or the code. But before that I wanted to ask if there would be an appetite to receive such changes. I'm not asking for a commitment to accept them 😅 , but just about whether you **think** now this has the **potential** for being integrated.

**Comment by bhou** [comment]
Thanks for bringing this up. You are more than welcome to contribute to the documentation. I think there are three types of audiences for the documentation:
- Users who use the command launcher to access commands
- Command Launcher package provider, who creates packages.
- Command Launcher administrator, who manages the registry, remote config, package release process etc.

At Criteo, we have our own implementation of the administration infrastructure; I feel that part is not very user-friendly for the public audience. Today's documentation structure might not be very obvious for the other two scenarios either. 

 Feel free to push PRs on the doc.

**Comment by hdost** [comment]
I think #150 was certainly part of this, are there further things you'd like to see on this @jdevera ?

**Comment by jdevera** [comment]
Yes, I stated that PR as a sort of superficial cleanup before I try to do something deeper. I'd like to leave this open for discussion if that's okay with you all.
